### ISSUE TITLE
numbers inside a string

### ISSUE THREAD

**Issue by Madrain** [TonyGravagno/MVBASIC-plus-plus#3]
a number inside a string gets treated as number if it was next to a delimiter 

![image](https://user-images.githubusercontent.com/30535720/28733516-32c7361a-73e5-11e7-9f1a-3489898c586c.png)

**Comment by TonyGravagno** [comment]
Yeah, I think I was over-aggressive about making numbers a keyword. I did that because older code uses numbers for labels and I wanted those labels to have the same colouring as text labels, which can include numbers. Otherwise the labels get the same funky color combinations too.

The solution to, now, that problem is to make labels look like another type of keyword. As you see, this is like a carpet with a loose thread .... pull out the thread and you need to pull more and eventually the carpet may unravel. But each individual now has the ability to pull on threads that we didn't have before, so pull away until you get a theme that works for you!

I have the revised XML definition that you sent and will test and then publish that as a new mvBase definition.

As I said in #2, a Real lexer/parser would be able to detect these conditions. I need to get through other projects before taking on the challenge of writing that kind of parser, but it's on my list of things I'd like to do here.
